### PR TITLE
Changelog entries for #16392 #16393 #16394 #16395

### PR DESCRIPTION
**Pull request by tomaszw**
### Link to issue number:
N/A

### Summary of the issue:
Changelog entries are missing for #16392 #16393 #16394 #16395

### Description of user facing changes
Added Changelog entries

### Description of development approach
N/A

### Testing strategy:
N/A

### Known issues with pull request:
N/A

### Code Review Checklist:

- [ ] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [ ] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [ ] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [ ] API is compatible with existing add-ons.
- [ ] Security precautions taken.

### CODE REVIEW

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -228,6 +230,11 @@ Users of Poedit 1 are encouraged to update to Poedit 3 if they want to rely on e
 - The speech text is no longer updated when the mouse moves in the Speech Viewer. (#15952, @hwf1324)
 - NVDA will again switch back to browse mode when closing combo boxes with ``escape`` or ``alt+upArrow`` in Firefox or Chrome. (#15653)
 - Arrowing up and down in combo boxes in iTunes will no longer inappropriately switch back to browse mode. (#15653)
+- Fixes related to HP Secure Browser support which also apply to other chromium-based browsers when in UIA mode
```
[the remark this answers is not in the document]
Comment (tomaszw) [reply user_docs/en/changes.t2t]: Thanks, moved to correct section and reworded as suggested

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -83,6 +85,11 @@ What's New in NVDA
 - Fixed a rare case when saving the configuration may fail to save all profiles.  (#16343, @CyrilleB79)
 -In Firefox and Chromium-based browsers,  NVDA will correctly enter focus mode when pressing enter when positioned within a presentational list (ul / ol) inside editable content. (#16325)
 - Column state change is automatically reported when selecting columns to display in Thunderbird message list. (#16323)
+- Fixes for Chromium-based browsers when used with UIA:
+  - Fixed NVDA hang when accessing some UIA documents after they're destroyed (#16393)
+  - Fixed NVDA hang when splitting some UIA text ranges (#16394)
+  - Fixed backspace key not working in GMail sign-in fields when using UIA (#16395)
```
Comment (CyrilleB79) [inline user_docs/en/changes.t2t:91]: A better and less technical wording. Feel free to ignore if you do not find it suitable.

Especially for last item, no need to mention "when using UIA" in the sub-item; we are already in an item dealing with Chromium when used with UIA.

```suggestion
  - Fixed bugs causing NVDA to hang. (#16393, #16394)
  - Backspace key is now working correctly in GMail sign-in fields. (#16395)
```

Comment (tomaszw) [reply user_docs/en/changes.t2t]: updated as such

**Comment by CyrilleB79** [comment]
Last remark, you may optionally add the reference to your GitHub's name along with the GitHub numbers mentioned in the change log.

It's not mandatory, but IMO, it help knowing who has worked on various parts of the code, what is sometimes useful. Just my opinion though.

File: user_docs/en/changes.t2t [hunk 3]
```diff
@@ -44,6 +44,8 @@ What's New in NVDA
   - The volume of the other applications can be adjusted by ``NVDA+alt+pageUp`` and ``NVDA+alt+pageDown``. (#16052, @mltony)
   - The sound of the other applications can be muted with ``NVDA+alt+delete``. (#16052, @mltony)
   -
+- HP Secure Browser is now supported (#16377)
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:47]:
```suggestion
- HP Secure Browser is now supported. (#16377)
```

File: user_docs/en/changes.t2t [hunk 4]
```diff
@@ -83,6 +85,10 @@ What's New in NVDA
 - Fixed a rare case when saving the configuration may fail to save all profiles.  (#16343, @CyrilleB79)
 -In Firefox and Chromium-based browsers,  NVDA will correctly enter focus mode when pressing enter when positioned within a presentational list (ul / ol) inside editable content. (#16325)
 - Column state change is automatically reported when selecting columns to display in Thunderbird message list. (#16323)
+- Fixes for Chromium-based browsers when used with UIA:
+  - Fixed bugs causing NVDA to hang. (#16393, #16394)
+  - Backspace key is now working correctly in GMail sign-in fields. (#16395)
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:90]:
```suggestion
  - Backspace key is now working correctly in Gmail sign-in fields. (#16395)
```

**Review by seanbudd — APPROVED** [review]
Thanks @tomaszw

**Pull request merged**